golang/main.go

```diff
@@ -1,7 +1,24 @@
 package main
 
-import "fmt"
+import (
+	"fmt"
+	"math"
+)
+
+func formula(x float64) float64 {
+	var result float64
+	result = math.Asin(math.Pow(x, 2)) + math.Asin(math.Pow(x, 3))
+	return result
```

Comment:
можно и сильно проще
```suggestion
	return math.Asin(math.Pow(x, 2)) + math.Asin(math.Pow(x, 3))
```